### PR TITLE
Add presentation for IDR

### PR DESCRIPTION
**Pull request by pwalczysko**
cc @jburel

### CODE REVIEW

**Comment by pwalczysko** [comment]
Do you want to add the link + screenshot  as in screenshot below to your presentation, saying that we are working towards this ?
Screenshot 2021-06-03 at 16 48 24

File: 2021/Dundee/IDR-workshop/index.html [hunk 1]
```diff
@@ -120,7 +131,50 @@ <h2>Search for images by term
...
+                            <li><a href="https://github.com/IDR/idr0093-mueller-perturbation" target="_blank">study: idr0093-mueller-perturbation</a>&nbsp;<a href="https://binder.bioimagearchive.org/v2/gh/IDR/idr0093-mueller-perturbation/master?urlpath=notebooks%2Fnotebooks%2Fidr0093_explore.ipynb%3FplateId%3D9283" target="_blank"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg" style="vertical-align:text-bottom"></a></li>
+                            <li><a href="https://github.com/IDR/idr0094-ellinger-sarscov2" target="_blank">study: idr0094-ellinger-sarscov2</a>&nbsp;<a href="https://binder.bioimagearchive.org/v2/gh/IDR/idr0094-ellinger-sarscov2/master?urlpath=notebooks%2Fnotebooks%2Fidr0094-ic50.ipynb%3FscreenId%3D2603" target="_blank"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg" style="vertical-align:text-bottom"></a></li>
+                        </ul>
+                    </section>
 
+                </section>
+                <section data-state="ome-theme">
+                    <h2>Today workshop</h2>
```
Comment (pwalczysko) [inline 2021/Dundee/IDR-workshop/index.html:171]: Today's workshop

File: 2021/Dundee/IDR-workshop/index.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+                            <li><a href="https://github.com/IDR/idr0093-mueller-perturbation" target="_blank">study: idr0093-mueller-perturbation</a>&nbsp;<a href="https://binder.bioimagearchive.org/v2/gh/IDR/idr0093-mueller-perturbation/master?urlpath=notebooks%2Fnotebooks%2Fidr0093_explore.ipynb%3FplateId%3D9283" target="_blank"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg" style="vertical-align:text-bottom"></a></li>
+                            <li><a href="https://github.com/IDR/idr0094-ellinger-sarscov2" target="_blank">study: idr0094-ellinger-sarscov2</a>&nbsp;<a href="https://binder.bioimagearchive.org/v2/gh/IDR/idr0094-ellinger-sarscov2/master?urlpath=notebooks%2Fnotebooks%2Fidr0094-ic50.ipynb%3FscreenId%3D2603" target="_blank"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg" style="vertical-align:text-bottom"></a></li>
+                        </ul>
+                    </section>
+
+                </section>
+                <section data-state="ome-theme">
+                    <h2>Today workshop</h2>
```
Comment (pwalczysko) [inline 2021/Dundee/IDR-workshop/index.html]:
```suggestion
                    <h2>Today's workshop</h2>
```

**Comment by pwalczysko** [comment]
As discussed, this https://github.com/ome/presentations/pull/148#issuecomment-853974892 is solved, will be just shown during the workshop.

**Comment by jburel** [comment]
Thanks @pwalczysko for spotting it,
Merging

**Pull request merged**